docs/interpretation_of_JSON_Schema.md

```diff
@@ -51,7 +51,7 @@ The following JSON Schema keywords are [merged](#Merging-models) with the alread
 ## Merging models
 Because of the recursive nature of the interpreter (and the nested nature of JSON Schema) it happens that two models needs to be merged together. 
 
-If only one side has a property defined, it is used as is, if both have it defined they are merged based on the following logic (look [here](./input_processing.md#Internal-model-representation) for more information about the CommonModel and its properties):
+If only one side has a property defined, it is used as is, if both have it defined they are merged based on the following logic (look  [here](./input_processing.md#Internal-model-representation) for more information about the CommonModel and its properties):
```

Comment:
this file is missing and I am not sure where it is supposed to point. 🤔

Reply:
That file probably was removed. We have to wait for @jonaslagoni when he comes back from vacation to answer us.